Python/thread_pthread.h

```diff
@@ -143,6 +143,7 @@ typedef struct {
 } pthread_lock;
 
 #define CHECK_STATUS(name)  if (status != 0) { perror(name); error = 1; }
+#define CHECK_STATUS_PTHREAD(name)  if (status != 0) { char* strerror_msg = strerror(status);  fprintf(stderr, "%s: %s\n", name, strerror_msg); }
```

Comment:
We use "Squash and merge" button always.
So there is no need to create "crean commit log" in pull request branch.

Would you add an entry to `Misc/NEWS` too?